src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+---
+templateKey: blog-post
+title: Deploying your application as a StarlingX Application
+author: Tomás N. P. Barros
+date: 
+category:
+  - label: Features & Updates
+---
+
+Deploying your application as a StarlingX application is the highest level of integration you can achieve with the platform. It is a step up from deploying an app via [Helm](https://www.starlingx.io/blog/starlingx-deploying-your-app-1/) or [FluxCD](https://www.starlingx.io/blog/starlingx-deploying-your-app-2/). In this post, I will guide you through the process of deploying the [demo app](https://github.com/bmuniz-daitan/poc-starlingx-messages) used in previous deployment posts, along with some basic management commands <!- more ->.
```

Comment:
I feel like we could do this introduction a little bit better. Take a look at the blog posts 1 and 2. First it is introduced all the ways that an application can be deployed and then it goes into the specifics of the method in the post. I also would add the type of configuration your stx have, like if it AIO or not, if it is virtual or bare metal and what release is using.

 Also maybe separate into a different section the information about the demo application

Reply:
I may include the other methods as for maintaining the pattern from the previous posts of the series, but I initially found that including this information again was not necessary.

I briefly mentioned the demo app, do you really think it is necessary to create a separate section about it?

Reply:
Maybe not a separate section, but I would definitely put it in a different paragraph.